packages/js/src/ui/components/Notification/NotificationList.tsx

```diff
@@ -13,7 +13,10 @@ export const NotificationListContainer = (props: ParentProps) => {
 
   return (
     <div
-      class={style('notificationList', 'nt-flex nt-flex-col nt-min-h-full nt-w-full nt-h-[37.5rem] nt-overflow-auto')}
+      class={style(
+        'notificationList',
+        'nt-flex nt-flex-col nt-min-h-full nt-min-w-full nt-w-96 nt-h-[37.5rem] nt-overflow-auto'
```

Comment:
Added a fixed width here until we decide how this will be determined. We need to specify the same width for skeleton and notification (and empty notification container) if we specify the width further down instead of here. @LetItRock thoughts?

Reply:
I'm fine with this.